scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -402,10 +401,42 @@ final case class BigQueryTypedTable[T: Coder](
         }
       )
       .pipe(w => Option(params.failedInsertRetryPolicy).fold(w)(w.withFailedInsertRetryPolicy))
-      .pipe(params.configOverride)
-      .pipe(w => if (params.extendedErrorInfo == Disabled) w else w.withExtendedErrorInfo())
+      .pipe(w => Option(params.configOverride).fold(w)(_(w)))
+      .pipe(w => if (params.extendedErrorInfo) w.withExtendedErrorInfo() else w)
+
+    val wr = data.applyInternal(transform)
+
+    var tuple = PCollectionTuple.empty(data.context.pipeline)
+    if (
```

Comment:
🎨  might be more readable as a `match` statement on `(params.method, data.internal.isBounded` - wdyt?